src/screens/VocabularyOverviewExerciseScreen.tsx

```diff
@@ -97,6 +105,15 @@ const VocabularyOverviewExerciseScreen = ({
     />
   )
 
+  if (error) {
```

Comment:
This is never shown for me. Tried
1. Open SubcategoryOverviewScreen
2. Disable WiFI
3. Click on a Subcategory
4. Exercise Overview is shown.

Reply:
For me it sometimes shows the error message and it sometimes still showss the endless loading spinner on this screen. However, I haven't yet figured out why it behaves like that.